src/centralized_telescope/proof.rs

```diff
@@ -0,0 +1,14 @@
+//! ALBA's Proof structure
+
+use crate::utils::types::Element;
+
+/// Alba proof
+#[derive(Debug, Clone)]
+pub struct Proof {
+    /// Numbers of retries done to find the proof
+    pub retry_counter: u64,
+    /// Index of the searched subtree to find the proof
+    pub search_counter: u64,
+    /// Sequence of elements from prover set
+    pub element_sequence: Vec<Element>,
```

Comment:
this vector apparently contains the proof. Should we give it a more descriptive name such as `proof_sequence`, `proof_elements` or `proof_content`?

Reply:
The proof not only comprises a sequence of elements of the prover set but also the counters, as such `proof_content` might be misleading.

Each of these parameters are within a proof, and will be accessed as `proof.search/retry_counter" and `proof.element_sequence`. Repeating "proof" in the variable name seems a bit redundant in that aspect.

The reason why @curiecrypt and I liked the word "sequence" is because of the implied notion of order that you wouldn't have in "set" or "list".